autogluon/task/tabular_prediction/predictor.py

```diff
@@ -178,7 +178,7 @@ def evaluate_predictions(self, y_true, y_pred, silent=False, auxiliary_metrics=F
         return self._learner.evaluate(y_true=y_true, y_pred=y_pred, silent=silent,
                                       auxiliary_metrics=auxiliary_metrics, detailed_report=detailed_report)
 
-    def leaderboard(self, dataset=None, silent=False):
+    def leaderboard(self, dataset=None, only_breakpoints=False, silent=False):
```

Comment:
consider clearer argument name instead of `only_breakpoints`, e.g. `only_top_models`.  May also consider mentioning somewhere that this only returns "models on the Pareto frontier of the accuracy/latency tradeoff"

Reply:
I didn't know the term Pareto frontier, but that describes it well! Agreed on only_top_models, the tricky part is what the definition of 'top model' is to a user, they might be confused if the 2nd highest acc model isn't present in that case.

Reply:
Renamed to only_pareto_frontier, added to documentation